content/doc/book/installing/_docker.adoc

```diff
@@ -21,28 +21,33 @@ _run-jenkins-in-docker.adoc).
 [source,bash]
 ----
 docker run \
-  -u root \
-  --rm \  # <1>
-  -d \ # <2>
-  -p 8080:8080 \ # <3>
-  -p 50000:50000 \ # <4>
-  -v jenkins-data:/var/jenkins_home \ # <5>
-  -v /var/run/docker.sock:/var/run/docker.sock \ # <6>
-  jenkinsci/blueocean # <7>
+  -u root \ # <1>
```

Comment:
How about this approach?
* creating `jenkins` user on the host that maps via uid to the jenkins user in the docker container
* `adduser jenkins docker`

Reply:
I don't want to complicate the instructions, just offer an explanation for why they are the way they are. I don't think it is recommended for the average user to create a `jenkins` user on the host and map it via uid and advanced users that are interested in doing that, probably know how to do that anyway.